tests/Aspire.Hosting.Keycloak.Tests/Aspire.Hosting.Keycloak.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there some sort of "functional test" that can be written? One that:

* Starts up the Keycloak container
* Makes requests to it like an app would
* Verifies that the requests return successful results

I know it isn't apples-to-apples, but here's that kind of test with a Redis container: 

https://github.com/dotnet/aspire/blob/cd6f0248191668209e6d37659982e97e52be46f9/tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs#L18-L52

Reply:
> Is there some sort of "functional test" that can be written? One that:
> 
> * Starts up the Keycloak container
> * Makes requests to it like an app would
> * Verifies that the requests return successful results
> 
> I know it isn't apples-to-apples, but here's that kind of test with a Redis container:
> 
> https://github.com/dotnet/aspire/blob/cd6f0248191668209e6d37659982e97e52be46f9/tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs#L18-L52

I guess a "functional" test for the Keycloak component would involve checking that one of the claims of the authenticated user can be retrieved from the UserPrincipal object, either from the minimal API side or the Blazor app side. 

In either case, the user has to first authenticate on Keycloak sign-in screen to complete the OIDC flow, which I'm not sure how to automate for the test.

Likely there's a way, but I can't allocate time right now to dive into it.